extensions/ql-vscode/src/extension.ts

```diff
@@ -792,12 +792,16 @@ async function activateWithInstalledDistribution(
   const languageSelectionPanel = new LanguageSelectionPanel(languageContext);
   ctx.subscriptions.push(languageSelectionPanel);
 
+  void extLogger.log("Initializing CodeQL language server.");
+  const languageClient = createLanguageClient(qlConfigurationListener);
```

Comment:
this changes the initialization order of things because we now inject the language client into sub-components. It seems to work fine for me locally.